### DISCUSSION TITLE
Q&A Thread

### DISCUSSION THREAD

**Discussion by TechnicallyCoded** [TechnicallyCoded/Inventory-Rollback-Plus#15] [Q&A]
## 👋 Welcome!
  This is a Q&A section where you can ask anyone for help regarding issues about the plugin, vaguely related to the plugin, or random help questions unrelated to this plugin.
  * Ask the questions you want.
  * Share ideas if you have answers or suggestions.
  * Welcome others and be open-minded. Remember that everyone here has a voice! Being negative will affect others, but so will being positive :)

  To get started, ask a question!

**Comment by oskarkk** [comment]
What does [pdd] mean?
![image](https://user-images.githubusercontent.com/16339637/139916420-9c0da09f-b19f-4bff-bc62-5c402632a576.png)

**Reply by TechnicallyCoded** [reply]
The plugin just copies from your inventory, which means your item already had that data on it, unless all items now have this magic text.. but I doubt it since I never heard of this problem

**Reply by oskarkk** [reply]
Okay, I found that it's from XItemsRemover plugin, which seems to add this to the lore of every item dropped on death.

Thanks for maintaining this great plugin 🙂

**Reply by TechnicallyCoded** [reply]
You found the solution to your issue, awesome! Thanks for the support ^^

**Comment by Tmdoofking** [comment]
I try most UTC +8 timezone name but it didnt work. (like CST, BNT)
what should i set for?

**Reply by TechnicallyCoded** [reply]
If the timezone is correct, it should display the timezone correctly in-game. There is a bug where the console will say it's invalid no matter what you enter so test it in-game.

**Comment by BharathASL** [comment]
Is there going to be an update for 1.21.9 anytime soon?

Thank you :)

**Reply by TechnicallyCoded** [reply]
Is this a joke lol?

**Reply by BharathASL** [reply]
It is just a genuine question to understand the ETA for the update. I didn’t mean it as a joke, so I’m not sure what was meant by that.

**Reply by BharathASL** [reply]
I recently started using this plugin and found it very helpful for our server. After the Copper Age update, we updated our server without checking the update status of the existing plugins, since we are still new to running a Minecraft server. I came across your GitHub and thought of asking if there are any plans for an update.

**Reply by TechnicallyCoded** [reply]
Wdym ETA lol? Just go download the update for 1.21.9 :P  It was made available 3 days ago.
